resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -2,9 +2,7 @@
 
 use App\Community\Enums\ArticleType;
 use App\Community\Enums\ClaimSetType;
-use App\Community\Enums\ClaimStatus;
 use App\Community\Enums\ClaimType;
-use App\Community\Enums\SubscriptionSubjectType;
```

Comment:
This still seems to be needed:
```
Class "SubscriptionSubjectType" not found
```

Reply:
Same for ClaimStatus:
```
Class "ClaimStatus" not found
```

Reply:
Thanks for catching that! Looks like pint was removing those on me. Added back.